### PR TITLE
interactive issue dependencies aka how/why diagram

### PR DESCRIPTION
**Pull request by dckc**
@Tartuffo @rowgraus @turadg It's deployed at https://agoric-eng-planning-1.netlify.app/
LMK if you want help with the API key stuff.

image

## Features

 - [x] manage API key credentials in localStorage
 - [x] show transitive dependencies
 - [x] show open/closed status as red/green
 - [x] support a release as the root
 - [x] deploy to netlify
 - ... more in #10

### CODE REVIEW

**Comment by dckc** [comment]
check it out at https://agoric-eng-planning-1.netlify.app/

File: planning/how-why/src/zenhubTools.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+export const queryWorkspace = async (id, endpoint) => {
+  const result = await endpoint.query(queries.workspace, { id });
+  if (!("data" in result)) {
+    throw Error("queryWorkspace: no data");
+  }
+  return result.data.workspace;
+};
+
+/** @param {Map<string, IssueInfo>} issueDetail */
+export const issueDepDot = (issueDetail) => {
```
Comment (turadg) [inline planning/how-why/src/zenhubTools.js:161]: the other exports of this package are for using the Zenhub API. consider `viz.js` or something like that for this one

**Review by turadg — APPROVED** [review]
Nothing blocking. I reviewed lightly since this is an internal tool. I assume review was requested mostly to ensure other people are acquainted with the design and implementation. 👍

**Comment by dckc** [comment]
todo idea: text rendering of the plan, suitable for use as a github task list

File: planning/how-why/src/app.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+// Initialize htm with Preact
+export const html = htm.bind(h);
+
+export const key = "zenhub.apikey";
+
+export const Agoric = {
+  repositories: {
+    ["agoric-sdk"]: 219012610,
+  },
+  workspaceId: "5fda3aa55dbe1100117cbaae", // XXX move to .html form value?
```
[the remark this answers is not in the document]
Comment (dckc) [reply planning/how-why/src/app.js:30]: it's configuration info. This code would work with any valid `workspaceId`.

File: planning/how-why/src/app.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+/** @typedef {import('./zenhubTools').IssueKey} IssueKey */
+/** @typedef {import('./zenhubTools').IssueInfo} IssueInfo */
+/** @typedef {import('./zenhubTools').IssueLeaf} IssueLeaf */
+
+// Initialize htm with Preact
+export const html = htm.bind(h);
+
+export const key = "zenhub.apikey";
+
+export const Agoric = {
```
[the remark this answers is not in the document]
Comment (dckc) [reply planning/how-why/src/app.js:26]: If I were going to change the code to be ignorant of whose configuration it is, I'd be inclined to move the configuration info to the .html UI.

**Pull request merged**